pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -48,6 +48,16 @@ func (m *kubeGenericRuntimeManager) createPodSandbox(pod *api.Pod, attempt uint3
 	return podSandBoxID, "", nil
 }
 
+// applyPodConfigAnnotations applies some pod configurations into annotations. These configurations
+// are usually experimental features. They'll be promoted to official runtime api in the future.
+func applySandboxConfigAnnotations(annotations map[string]string, pod *api.Pod) {
```

Comment:
I see. If we assume `pod.Annotations` should be passed to runtime, we can remove this.